src/ar100/Makefile

```diff
@@ -0,0 +1,45 @@
+CROSS_PREFIX=or1k-linux-musl-
+dirs-y += src/generic src/ar100 lib/ar100
+
+CFLAGS		+= -O3
```

Comment:
Lets keep this PR as it is.  But, out of curiosity, have you tried `-Os`?  It was a notable improvement in the PRU code size.  To avoid a (relatively minor) performance hit on the PRU I also added `__attribute__((optimize("O2")))` to `irq_poll(void)` on that mcu - see src/pru/main.c .

-Kevin

Reply:
Oh, very nice! For some reason the code generated is quite large. It would be great to optimize the parts involved in stepping for speed, but the rest for size. I'll create a ticket for it in my fork and take a look.